### PR TITLE
Feature/draft 5

### PR DESCRIPTION
**Pull request by ivaylomitrev**

### CODE REVIEW

File: v1/examples.md [hunk 1]
```diff
@@ -651,3 +652,73 @@ Response:
...
+```
+POST /document/search HTTP/1.1
+Authorization: Bearer {token}
+Content-type: application/json
+
+{
+    "query": "Termination date",
+    "queryAttributes": ["documentVersions.text"],
+    "filterQuery": "entry.section.title=@sectionTitle && (entry.tags.title=@tagTitle1 || entry.tags.title=@tagTitle2)",
+    "parameters": {"@sectiontitle": "HR", "@tagTitle1": "Contracts", "@tagTitle2": "John Doe"},
```
Comment (phillipj) [inline v1/examples.md]: Are the parameters case sensitive? Ref the difference of `@sectiontitle | @sectionTitle` in the parameters and the filter query.

File: v1/free-text-search-query-language.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
+Free-text search query language
+-------
+
+## Overview
+
+The web services support a simple but powerful free-text search language that allows one to searh both in the metadata and content of documents uploaded to Documaster. Additional **criteria** can be specified on the **resource attributes** or on the **attributes of linked resources** to narrow down the results. Note that a *linked resource* may be linked directly or indirectly (via another resource).
```
Comment (phillipj) [inline v1/free-text-search-query-language.md]: `searh -> search`

**Review by phillipj — APPROVED** [review]
Other than a few nitpicks, this LGTM.

File: v1/examples.md (hunk 1 above)
Comment (ivaylomitrev) [reply v1/examples.md]: Definitely not. It's a typo. Fixed.

File: v1/free-text-search-query-language.md (hunk 2 above)
Comment (ivaylomitrev) [reply v1/free-text-search-query-language.md]: Good catch. Fixed.

**Comment by ivaylomitrev** [comment]
Squashed in preparation for merging.

**Pull request merged**